20230727/s8-1690457241591/README.md

```diff
@@ -0,0 +1,62 @@
+# Scenario 8 - ALS Baseline with Sims-only, multiple k6 VUs
```

Comment:
The K6 VU count is dynamic, not static?

Reply:
There was no substantial difference between 1-5, thus why we only captured one of the results

Reply:
Also fixed title.